src/runtime_config/default/actions.js

```diff
@@ -1060,4 +1060,29 @@ const actions = new Map([
       }
     ),
   ],
+  [
+    "migrate_service_to_2tx_reconfig",
```

Comment:
Should this just be a generic `set_service_configuration` action that can have an arbitrary number of fields as arguments and would update those and only those? The existing `set_recovery_threshold` action could be removed to use this too.

Reply:
Yeah, I think in the long run that's the right thing to do, just offer one general way to perform config updates.